src/test/kotlin/com/igorwojda/integer/stepsgenerator/challenge.kt

```diff
@@ -4,7 +4,15 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 fun generateSteps(n: Int): List<String> {
-    TODO("not implemented")
+    val listOfHash = mutableListOf<String>()
+    for (i in 1..n) {
+        var line = ""
+        for (j in 1..n) {
+            line += if (i >= j) '#' else ' '
```

Comment:
I know it is very tempting to write such statement on a single line, but I recommend using curly braces for the if-else statements. It makes code easier to read and to debug.